lib/active_model/serializer/adapter/json_api/relationship.rb

```diff
@@ -0,0 +1,53 @@
+module ActiveModel
+  class Serializer
+    module Adapter
+      class JsonApi
+        class Relationship
```

Comment:
@bf4 following your [comment](https://github.com/rails-api/active_model_serializers/pull/1504/files#r52563899) I was not sure if they should still be namespace by `ApiObjects` or not.